### ISSUE TITLE
Status page for JuliaHub and related web services

### ISSUE THREAD

**Issue by DilumAluthge** [JuliaComputing/JuliaHub#63]
We now have a great status page for open-source JuliaLang services: https://status.julialang.org/

Perhaps Julia Computing could set up a similar status page for JC websites, such as:
- Julia Computing website
- JuliaHub website
- JuliaHub Registrator web UI instance

**Comment by DilumAluthge** [comment]
Bump. Any further thoughts on this?

Probably the most useful statuses will be:
1. Registrator GitHub app
2. JuliaHub.com Registrator web UI

cc: @StefanKarpinski @ViralBShah

**Comment by DilumAluthge** [comment]
Actually, now that I think about it, since those two are offered by JC as a free public service to the community, we could probably just put those two status on https://status.julialang.org

But the other stuff (JC website, pkg.juliahub.com, etc.) will definitely be better suited for a JC-specific status page.

**Comment by ViralBShah** [comment]
Yes, registrator makes sense - but status has been down more than the stuff it monitors.

**Comment by PallHaraldsson** [comment]
I'm not sure it's broken, I see:
```
Homepage and Forums
Package Servers
Storage Servers
Build Infrastructure
```

Should it say more, "up" or "down" after?

**Comment by DilumAluthge** [comment]
Yeah it's just show a bunch of up time and downtime information
